pypeit/telescopes.py

```diff
@@ -147,6 +147,15 @@ def __init__(self):
                                                eff_aperture=11.2)
 # KPNO from https://en.wikipedia.org/wiki/Nicholas_U._Mayall_Telescope
 
+class HiltnerTelescopePar(TelescopePar): 
+    def __init__(self):
+        super(HiltnerTelescopePar, self).__init__(name='HILTNER',
+                                              longitude=111.615556,           # Longitude of the telescope (NOTE: West should correspond to positive longitudes)
```

Comment:
This was the old way of doing it... and so there's a subtle bug here. Astropy changed it's definition to be positive for EAST not WEST... so this should be -111.615556, but it would be much better to do the following:

```
class KPNOTelescopePar(TelescopePar):
    def __init__(self):
        loc = EarthLocation.of_site('mdm')
        super(KPNOTelescopePar, self).__init__(name='HILTNER',
                                               longitude=loc.lon.to(units.deg).value,
                                               latitude=loc.lat.to(units.deg).value,
                                               elevation=loc.height.to(units.m).value,
                                               diameter=2.4)
```

Reply:
Good catch, fixed!